src/cart/crt_init.c

```diff
@@ -348,6 +348,9 @@ static int data_init(int server, crt_init_options_t *opt)
 	}
 	if (server)
 		d_setenv("UCX_IB_FORK_INIT", "n", 1);
+	else
+		d_setenv("UCX_IB_FORK_INIT", "y", 1);
```

Comment:
didnt we want to set IBV_FORK_SAFE one instead? i thought setting UCX ones caused runtime issues with some file not being found

Reply:
https://github.com/daos-stack/daos/pull/13991 was also submitted to try with IBV_FORK_SAFE setting. (The Jira ticket has a comment stating that I'm trying both approaches.)

Reply:
Both of these PRs are drafts.  I would prefer the UCX settings if it's possible to make them work.

Reply:
Cool, sounds good, thanks